controllers/placementrule/placementrule_controller.go

```diff
@@ -122,9 +126,10 @@ func (r *PlacementRuleReconciler) Reconcile(ctx context.Context, req ctrl.Reques
 	}
 
 	//read image manifest configmap to be used to replace the image for each component.
-	if _, err = config.ReadImageManifestConfigMap(r.Client); err != nil {
-		return ctrl.Result{}, err
-	}
+	// the MultiClusterObservabilityReconciler will read the image manifests, no need to read it again.
```

Comment:
to delete readimage logic maybe lead to problem
at starting stage
if placementrule controller reconcile firstly, then there is no image info and that will lead to wrong images used in the manifestwork
The images will not be updated until next reconcile.
in worst cases, the wrong images will be kept for a long time

Reply:
Good catch.

Reply:
I added the read image logic back.